packages/components/src/list/collapsible-list/index.tsx

```diff
@@ -12,32 +12,62 @@ import { ExperimentalListItem } from '../experimental-list-item';
 import { ListProps, ExperimentalList } from '../experimental-list';
 
 type CollapsibleListProps = {
-	hideText: string;
-	showText: string;
+	collapseLabel: string;
+	expandLabel: string;
 	collapsed?: boolean;
-	minChildrenToShow?: number;
+	show?: number;
 	onCollapse?: () => void;
 	onExpand?: () => void;
 } & ListProps;
 
-const TASK_HEIGHT = 70;
+const FALLBACK_TASK_HEIGHT = 70;
 const defaultStyle = {
 	transition: `max-height 500ms ease-in-out`,
 	maxHeight: 0,
 	overflow: 'hidden',
 };
 
+function getContainerHeight(
+	childrenLength: number,
+	collapseContainer: HTMLDivElement | null
+) {
+	let containerHeight = 0;
+	if ( collapseContainer ) {
+		for ( const child of collapseContainer.children ) {
+			containerHeight += child.clientHeight;
+		}
+	}
+	if ( containerHeight === 0 ) {
+		containerHeight = childrenLength * FALLBACK_TASK_HEIGHT;
```

Comment:
As discussed in Slack, I think it's safe to remove this (certainly until we have something reproducible exemplifying it's necessity).